cli/integration-test/integration/fixture.clj

```diff
@@ -14,11 +14,17 @@
 
 (defn initialize-request
   ([]
-   (initialize-request {:lint-project-files-after-startup? false}))
-  ([settings]
+   (initialize-request {:initializationOptions {:lint-project-files-after-startup? false}}))
+  ([params]
    (lsp-json-rpc :initialize
-                 {:rootUri (h/file->uri (io/file h/root-project-path))
-                  :initializationOptions settings})))
+                 (merge {:rootUri (h/file->uri (io/file h/root-project-path))}
+                        params))))
+
+(defn completion-request [path row col kind]
+  (lsp-json-rpc :textDocument/completion
+                {:textDocument {:uri (h/file->uri (h/source-path->file path))}
+                 :position {:line row :character col}
+                 :context {:triggerKind kind}}))
```

Comment:
Do we need to send this :context or triggerKind? we don't check for that any time on clojure-lsp

Reply:
I didn't check if we use it, just that it's getting sent. I'll remove it.